tests/templates/test_subroutines/test_prepselprep.py

```diff
@@ -32,10 +31,19 @@
         (qml.Hamiltonian([0.25, 0.75], [qml.Z(2), qml.X(1) @ qml.X(2)]), [0]),
         (0.25 * qml.Z(2) - 0.75 * qml.X(1) @ qml.X(2), [0]),
         (qml.Z(2) + qml.X(1) @ qml.X(2), [0]),
-        (qml.ops.LinearCombination([-0.25, 0.75j], [qml.Z(3), qml.X(2) @ qml.X(3)]), [0, 1]),
-        (
+        pytest.param(
+            qml.ops.LinearCombination([-0.25, 0.75j], [qml.Z(3), qml.X(2) @ qml.X(3)]),
+            [0, 1],
+            marks=pytest.mark.skip(
+                reason="This test case is expected to fail due to `NaN`s being returned in the gradient. This is a known limitation of ``MottonenStatePreparation``."
+            ),
```

Comment:
Instead of skipping `assert_valid` completely for this test case, you should pass `skip_differentiation=True` to the `assert_valid` call for this particular test case.

Reply:
Ah yes, forgot about that argument. Thanks! 👍🏼